<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyDialog</class>
 <widget class="QDialog" name="modifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,137,216)</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b_21</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b_21</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>04b_21</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>输入原密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>输入新密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>确认新密码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close1Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#close1Button
{
	background:transparent
}
QPushButton#close1Button
{	
    border-image: url(:/Resources/images/close.png);
}
QPushButton#close1Button:hover
{
	border-image: url(:/Resources/images/close_hover.png);
}
QPushButton#close1Button:pressed
{
    border-image: url(:/Resources/images/close_press.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#okButton
{
	font: 25 9pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#okButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:2px;
    padding-top:2px;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#cancelButton
{
	font: 25 9pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#cancelButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:2px;
    padding-top:2px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
